<p>It exits your method because you have a return statement inside of your catch block. You are telling it to return explicitly.</p>
